### PR TITLE
OKTA-677403 : Removing password requirement icon title

### PR DESCRIPTION
**Pull request by glenfannin-okta**
## Description:

The purpose of this PR is to remove the "title" property from the Password Requirement status icons. These titles were causing localization failures in our test suites but after some analysis, it was deemed that these are not needed since they are hidden from screen readers and since for there is no benefit from the title that can be portrayed to visual users. 

This PR also includes a minor UI fix to the phone authenticator component, removing an unnecessary space.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-677403](https://oktainc.atlassian.net/browse/OKTA-677403)

### Screenshot/Video:
Axe dev tool scan: 
Screenshot 2023-12-19 at 6 49 30 PM

This PR also includes a minor UI fix for the phone authenticator component (removes an unnecessary space when the extension is added to the view for voice calls): 

Before Fix (notice extra space):

https://github.com/okta/okta-signin-widget/assets/97472729/f01570dc-ba30-4d7b-876c-fa2e6c6dee2b

After Fix: 

https://github.com/okta/okta-signin-widget/assets/97472729/49e28a5e-38aa-477e-8d70-50e36e25567e

### CODE REVIEW

File: src/v3/src/types/schema.ts [hunk 1]
```diff
@@ -240,7 +240,7 @@ export interface UISchemaLayout extends Layout {
 
 export interface StepperLayout extends Layout {
   type: UISchemaLayoutType.STEPPER;
-  elements: Omit<UISchemaLayout, 'StepperLayout'>[];
+  elements: Omit<UISchemaLayout['elements'], 'StepperLayout'>;
```
Comment (trevoring-okta) [inline src/v3/src/types/schema.ts:243]: Is this another small throw-in change?

Comment (glenfannin-okta) [reply src/v3/src/types/schema.ts:243]: lol yea, this doesn't functionally impact anything but when using types, this is the proper way to model this interface to exclude the stepper from nesting itself.

File: src/v3/src/components/Icon/CheckCircle.tsx [hunk 2]
```diff
@@ -23,7 +23,6 @@ export const CheckCircle: FunctionComponent<IconProps> = ({ description }) => (
     xmlns="http://www.w3.org/2000/svg"
     role="img"
   >
-    <title>{description}</title>
     <path
```
Comment (trevoring-okta) [inline src/v3/src/components/Icon/CheckCircle.tsx:26]: Not familiar with SVG `title`, does it not appear in the DOM so no snapshots need to be updated?

Comment (glenfannin-okta) [reply src/v3/src/components/Icon/CheckCircle.tsx:26]: Oh yea, snapshots will need to be updated :( committing that now

**Review by trevoring-okta — APPROVED** [review]

File: src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator-with-skip.test.tsx.snap [hunk 3]
```diff
@@ -490,7 +490,7 @@ exports[`authenticator-enroll-select-authenticator renders form 1`] = `
                       tabindex="0"
                       type="submit"
                     >
-                      Set up later
+                      Continue
```
Comment (glenfannin-okta) [inline src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator-with-skip.test.tsx.snap:493]: The i18n key was updated months ago, not sure why this test was never updated, nor do I understand how it ever passed 😅

**Pull request merged**